### DISCUSSION TITLE
Table: Allow setting a column is hover text

### DISCUSSION THREAD

**Discussion by amnonh** [grafana/grafana#38236] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
When you hover over a table cell, allow to configure the hover text from a different hidden cell.

**Why is this needed**:
The issue displaying a text that is larger than a table cell size is common.
There are multiple solutions, but one of the limitation, is that cutting the text (or using scroll for that purpose) make a table unreadable when looking at a dashboard (think about a kiosk mode).

### My use case for example
I'm using a a table to show alerts that I get from Loki, I can set a small description for the error that would fit nicely in a table cell
and I can place the entire log message in a different label.
This way, the text that users will see when looking at the table will be short but meaningful and hover over the text will give users the full information

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
